### ISSUE TITLE
Svelte 5 Dependency

### ISSUE THREAD

**Issue by sst-sat** [simeydotme/svelte-range-slider-pips#155]
Currently it is not possible (without a workaround) to use this lib in a svelte 5 project.

My current workaround is to add `|| ^5.0.0-next.0` in my package-lock.json.

**Comment by simeydotme** [comment]
huh?

**Comment by simeydotme** [comment]
it was working before Svelte 5 launched;
https://github.com/simeydotme/svelte-range-slider-pips/tree/main/tests/svelte5

**Comment by simeydotme** [comment]
I'll see if their full release broke something, you have any hints / screenshots or error codes to share?

**Comment by mihrich** [comment]
Hi boys!

modules with the latest updates are not installed (npm install):

```
{
  ...
  "devDependencies": {
    "@sveltejs/adapter-auto": "3.3.1",
    "@sveltejs/adapter-node": "5.2.9",
    "@sveltejs/kit": "2.7.3",
    "svelte": "^5.1.6",
    "svelte-range-slider-pips": "^3.1.2"
  }
  ...
}
```
![2024-10-31_18-43-59](https://github.com/user-attachments/assets/f9e10be1-22d4-4ba1-9492-67d2a0ad05c2)

**Comment by sst-sat** [comment]
Same problem like @mihrich .

It works in the test because you are using the code directly without pulling the dependency from npm.

**Comment by burtonrj** [comment]
Having the same issue as @mihrich

@sst-sat would you mind sharing more details RE your workaround? I added `|| ^5.0.0-next.0` to my package.json file but I'm still getting a dependency error from npm

**Comment by simeydotme** [comment]
thanks for this 

> Hi boys!
> 
> package.json with latest libs:
> 
> ```
> {
>   ...
>   "devDependencies": {
>     "@sveltejs/adapter-auto": "3.3.1",
>     "@sveltejs/adapter-node": "5.2.9",
>     "@sveltejs/kit": "2.7.3",
>     "svelte": "^5.1.6",
>     "svelte-range-slider-pips": "^3.1.2"
>   }
>   ...
> }
> ```
> 
> after **npm install** command:
> 
> ![2024-10-31_18-43-59](https://private-user-images.githubusercontent.com/1442402/382013649-f9e10be1-22d4-4ba1-9492-67d2a0ad05c2.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzA2ODgyNjQsIm5iZiI6MTczMDY4Nzk2NCwicGF0aCI6Ii8xNDQyNDAyLzM4MjAxMzY0OS1mOWUxMGJlMS0yMmQ0LTRiYTEtOTQ5Mi02N2QyYTBhZDA1YzIucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI0MTEwNCUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNDExMDRUMDIzOTI0WiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9OTg0Y2RjNTIxMmFmODFjZmYxZGMwM2ExMmI3MWMyODE3ZTYwM2RjNmZmOWU5NjY0OTU3YWQxMTBmOGY5M2Y2NSZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QifQ.eAlSx5-xcxXuVl7ux7OBhJspauilRceDN-NLD2QMgns)

I'll try to fix it tonight :)

**Comment by simeydotme** [comment]
> Currently it is not possible (without a workaround) to use this lib in a svelte 5 project.
> 
> My current workaround is to add `|| ^5.0.0-next.0` in my package-lock.json.

thanks for the issue, should be resolved with [ `3.1.4`](https://github.com/simeydotme/svelte-range-slider-pips/releases/tag/3.1.4) :)
